addons/medical/functions/fnc_handleDamage_advanced.sqf

```diff
@@ -23,12 +20,23 @@
 
 #include "script_component.hpp"
 
-params ["_unit", "_selectionName", "_amountOfDamage", "_sourceOfDamage", "_typeOfProjectile", "_hitPointNumber", "_newDamage"];
+params ["_unit", "_selectionName", "_amountOfDamage", "_sourceOfDamage", "_typeOfProjectile", "_hitPointNumber", "", "_newDamage"];
 
-//Temp fix for 1.63 handleDamage changes
-if (_newDamage isEqualType objNull) then {
-    _newDamage = _this select 7;
-};
+// For burning damage we will get a ton of very small hits of damage; they are too small to create any wounds
+// Save them up in a variable and run when it is over a noticable amount
+
+if ((_typeOfProjectile == "") && {_newDamage < 0.15} && {
```

Comment:
Are we sure that fire is the only damage source that matches this condition? iirc falling also gives you an empty projectile string. Same with being driven over and a few other cases.